### PR TITLE
fix Index mapping curl to include necessary Content-Type Header for P…

### PR DESCRIPTION
**Pull request by shakalandy**
- fix Index mapping curl to include necessary Content-Type Header for PUT requests.
- include link to ES documentation on index mapping

### CODE REVIEW

File: pages/configuration/elasticsearch.rst [hunk 1]
```diff
@@ -478,6 +478,7 @@ After you've removed the index template, new indices will only have the original
     }
   }
 
+Additional information on Elasticsearch Index Templates can be found in the official `Elasticseach Documentation <https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html>`:
```
Comment (jalogisch) [inline pages/configuration/elasticsearch.rst]: would you please write the link following:

```
`Elasticseach Documentation <https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html>`_
```

In addition we refer linking to a specific version and not current as Elastic tend to change how the product work between versions and we mostly refer to the latest 'official' supported version in Graylog. What is 6.6 at the time of writing.

File: pages/configuration/elasticsearch.rst [hunk 2]
```diff
@@ -338,7 +338,7 @@ Save the following index template for the custom index mapping into a file named
 
 Finally, load the index mapping into Elasticsearch with the following command::
 
-  $ curl -X PUT -d @'graylog-custom-mapping.json' 'http://localhost:9200/_template/graylog-custom-mapping?pretty'
+  $ curl -X PUT -d @'graylog-custom-mapping.json' -H 'Content-Type: application/json' 'http://localhost:9200/_template/graylog-custom-mapping?pretty'
```
Comment (jalogisch) [inline pages/configuration/elasticsearch.rst:341]: thank you for this catch. 

Internally we moved to using [httpie](https://httpie.org/) that is why such missing bits are unseen.

**Review by jalogisch — CHANGES_REQUESTED** [review]
LGTM just one small change is needed.

**Comment by jalogisch** [comment]
@shakalandy in addition you would need to sign the CLA that we can merge your contribution to the documentation.

File: pages/configuration/elasticsearch.rst (hunk 1 above)
Comment (shakalandy) [reply pages/configuration/elasticsearch.rst]: Updated the link ( and some more to 6.6) hope that is ok.

**Comment by shakalandy** [comment]
I signed the CLA already, seems that my initial push was done with my local git user, and the CLA is not updated after changing to the right "GitHub" one - sorry, i'm a n00b :)

**Comment by jalogisch** [comment]
thank you @shakalandy it might sound childish, but did you mind changing the author of the commit so we can merge?

This might help with that: https://www.git-tower.com/learn/git/faq/change-author-name-email

We can only merge if the CLA check is green, what indicates that each author has signed the CLA, but that is connected to your github handle.

**Comment by jalogisch** [comment]
on request this is closed.

**Pull request closed without merging**